src/mc_observers/EncoderObserver.cpp

```diff
@@ -70,8 +70,15 @@ void EncoderObserver::configure(const mc_control::MCController & ctl, const mc_r
     ;
   }
 
-  config("logPosition", logPosition_);
-  config("logVelocity", logVelocity_);
+  config("computeFK", computeFK_);
+  config("computeFV", computeFV_);
```

Comment:
Yes, efficiency when you know what you're doing ;)
The thing is as-is, there are cases where calling FK/FV in each controller, and if you know they're not needed it makes sense to skip it. For instance `EncoderObserver -> BodySensorObserver (FloatingBase)`, as the `FloatingBase` sensor is rigidly attached to the floating base, this observer does not need an up-to date FK before running, and it will compute the full FK itself when setting the estimated floating base pose. That's a bit of an expert feature for when you really know what you're doing (hence the default value to `true`). I'll add a disclaimer in the schema.

A better solution would probably be a nicer update mechanism, which might become more feasible with TVM.